src/components/DirectLinks/DirectLinks.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './DirectLinks.module.scss';
+import { LINKS } from './DirectLinks.constants';
+import { ReactComponent as OutsideLink } from 'assets/icons/Outside-Link.svg';
+
+export default function DirectLinks() {
+  return (
+    <div className={styles.DirectLinks}>
+      <div className={styles.InnerContainer}>
+        <h2>Direct naar</h2>
+        <ul className={styles.LinkList}>
+          {Object.keys(LINKS).map(link => {
+            const { url, displayName } = LINKS[link];
```

Comment:
I've used `title` instead of `displayName` for link titles / headings in other places, it's more common. Can you align that?

Reply:
OK, changed it